### PR TITLE
Implement SwapMode StandardReverse

### PR DESCRIPTION
**Pull request by greggameplayer**
Used by CUSA00006 (Knack)

I'm not sure if the implementation is 100% good

### CODE REVIEW

**Comment by Graubek** [comment]
Knack have new output in console
v0.2.0 
```
[Debug] <Critical> liverpool.cpp:ProcessGraphics:180: Unreachable code!
Invalid PM4 type 0
```
this commit
```
[Debug] <Critical> liverpool.cpp:ProcessGraphics:183: Unreachable code!
Unsupported PM4 type 0
```
[shad_log.txt](https://github.com/user-attachments/files/16813973/shad_log.txt)

**Comment by greggameplayer** [comment]
@Graubek according to my multiple tests this happens not all the time but sometimes, i don't think it's related to this PR

File: src/video_core/renderer_vulkan/liverpool_to_vk.cpp [hunk 1]
```diff
@@ -602,6 +605,21 @@ vk::Format AdjustColorBufferFormat(vk::Format base_format,
         default:
             break;
         }
+    } else if (comp_swap_reverse) {
+        switch (base_format) {
+        case vk::Format::eR8G8B8A8Unorm:
+            return vk::Format::eA8B8G8R8UnormPack32;
+        case vk::Format::eB8G8R8A8Unorm:
+            return vk::Format::eA8B8G8R8UnormPack32;
```
Comment (raphaelthegreat) [inline src/video_core/renderer_vulkan/liverpool_to_vk.cpp:613]: This should be A8R8G8A8. There is no available format so should be removed from switch

Comment (greggameplayer) [reply src/video_core/renderer_vulkan/liverpool_to_vk.cpp:613]: Okay that confirms what i was thinking, i will do it after my holiday trip

**Comment by Emulator-Team-2** [comment]
@greggameplayer 
Can you also add SwapMode - `AlternateReverse` used by Virtua Fighter 5 Ultimate Showdown.

**Comment by georgemoralis** [comment]
done similar in #814

**Pull request closed without merging**